internal/runtime/hcsv2/process.go

```diff
@@ -103,6 +103,13 @@ func newProcess(c *Container, spec *oci.Process, process runtime.Process, pid ui
 		// least one waiter has read the result
 		go func() {
 			p.writersWg.Wait()
+			// cleanup the process state
```

Comment:
Just to make sure I'm following from a quick skim, we'll essentially fall into here when the process is finished but the problem is we can't wait to clean up the process state at container stop time as this issue is for long running containers that might be spawning a bunch of processes during it's lifetime?

Reply:
That's correct. All these directories will automatically get cleaned up when the container stops but for long running containers that doesn't work.

Reply:
Gotcha ok 👍